opentracing-api/src/main/java/io/opentracing/Tracer.java

```diff
@@ -125,8 +151,54 @@
         /** Specify a timestamp of when the Span was started, represented in microseconds since epoch. */
         SpanBuilder withStartTimestamp(long microseconds);
 
-        /** Returns the started Span. */
-        Span start();
+        /**
+         * Returns a newly started and activated {@link ActiveSpan}.
+         *
+         * <p>
+         * The returned {@link ActiveSpan} supports try-with-resources. For example:
+         * <pre>{@code
+         *     try (ActiveSpan span = tracer.buildSpan("...").startActive()) {
+         *         // (Do work)
+         *         span.setTag( ... );  // etc, etc
+         *     }  // Span finishes automatically unless pinned via {@link ActiveSpan#capture }
+         * }</pre>
+         *
+         * <p>
+         * If
+         * <ul>
+         * <li>the {@link Tracer}'s {@link ActiveSpanSource#activeSpan()} is not null, and
+         * <li>no <b>explicit</b> references are added via {@link SpanBuilder#addReference}, and
+         * <li>{@link SpanBuilder#ignoreActiveSpan()} is not invoked,
+         * </ul>
+         * ... then an inferred {@link References#CHILD_OF} reference is created to the
+         * {@link ActiveSpanSource#activeSpan()} {@link SpanContext} when either {@link SpanBuilder#startManual()} or
+         * {@link SpanBuilder#startActive} is invoked.
+         *
+         * <p>
+         * Note: {@link SpanBuilder#startActive()} is a shorthand for
+         * {@code tracer.makeActive(spanBuilder.startManual())}.
+         * </p>
+         *
+         * @return a pre-activated {@link ActiveSpan}
+         *
+         * @see ActiveSpanSource
+         * @see ActiveSpan
+         */
+        ActiveSpan startActive();
 
+        /**
+         * Like {@link #startActive()}, but the returned {@link Span} has not been activated per
+         * {@link ActiveSpanSource}.
+         *
+         * @see SpanBuilder#startActive()
+         * @return the newly-started Span instance, which has *not* been automatically activated by the
+         *         {@link ActiveSpanSource}
+         */
+        Span startManual();
```

Comment:
Yeah, `finish(ts)` is something I'd only expect when a Span is managed manually.